CMakeLists.txt

```diff
@@ -39,27 +43,29 @@ mxnet_option(INSTALL_EXAMPLES     "Install the example source files." OFF)
 mxnet_option(USE_SIGNAL_HANDLER   "Print stack traces on segfaults." OFF)
 
 if(USE_CUDA AND NOT USE_OLDCMAKECUDA)
-  message(STATUS "CMake version '${CMAKE_VERSION}' using generator '${CMAKE_GENERATOR}'")
```

Comment:
can we leave the diagnostics about cmake version and generator?

Reply:
I moved it to log unconditionally at the top of the file